lib/std/enums.zig

```diff
@@ -1314,10 +1313,20 @@ pub fn EnumIndexer(comptime E: type) type {
             }
         };
     }
-    std.mem.sort(EnumField, &fields, {}, ascByValue);
-    const min = fields[0].value;
-    const max = fields[fields.len - 1].value;
-    const fields_len = fields.len;
+
+    const SortContext = struct {
+        fields: []EnumField,
+
+        pub fn lessThan(comptime ctx: @This(), comptime a: usize, comptime b: usize) bool {
+            return ctx.fields[a].value < ctx.fields[b].value;
+        }
+
+        pub fn swap(comptime ctx: @This(), comptime a: usize, comptime b: usize) void {
+            return std.mem.swap(EnumField, &ctx.fields[a], &ctx.fields[b]);
+        }
+    };
+    std.sort.insertionContext(0, fields_len, SortContext{ .fields = &fields });
```

Comment:
Why is this switched to insertion sort?

Reply:
reduce branches. otherwise we need to set `@setEvalQuotaBrnach(1500~2000)` due to safety check change.